pkg/util/utils.go

```diff
@@ -56,6 +57,52 @@ func parseCreds(creds string) (string, string) {
 	return up[0], up[1]
 }
 
+// Takes build context and validates `.containerignore` or `.dockerignore`
+// if they are symlink outside of buildcontext. Returns list of files to be
+// excluded and resolved path to the ignore files inside build context or error
+func ParseDockerignore(root string) ([]string, string, error) {
```

Comment:
Should we be using imagebuilder.ParseDockerignore?

Reply:
It only returns list of `excludes` for `podman-remote` and `podman-server` we need to do symlink validation as well. We can add a function there to do symlink validate as well but it will make PR hard to backport.